infra/variables.tf

```diff
@@ -29,6 +29,7 @@ variable "region" {
 
 # HSA
 
+# tflint-ignore: terraform_unused_declarations
 variable "zone" {
   type        = string
   description = "GCP zone for provisioning zonal resources."
```

Comment:
suggestion: Since this is a no-op and we may want to remove the use of zone from documentation/etc before we remove here, maybe we should add a default so we can ignore this variable.